src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1068,10 +1073,15 @@ void cbmc_parse_optionst::help()
     "Java Bytecode frontend options:\n"
     " --classpath dir/jar          set the classpath\n"
     " --main-class class-name      set the name of the main class\n"
+    " --java-assume-inputs-non-null test harness makes input arguments not null\n"
+    " --java-max-input-array-length N maximum allowed length for an array passed as input\n"
+    " --java-max-input-tree-depth N   object references are (deterministically) set to null in the object\n"
```

Comment:
These warnings are now fixed. I added 1 single commit (24e9dde) and while Travis will rebuild everything, the only changes are those in that commit, which are only C++ comments.

Can we thus skip Travis and merge? If possible I would like to avoid rebasing this one more time :)